src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -128,7 +128,7 @@ const CanonicalRadioSchedule = ({ endpoint }) => {
         .catch(e => logger.error(`HTTP Error: "${e}"`));
 
     fetchRadioScheduleData(endpoint);
-  }, [endpoint, service, script, timezone, locale]);
+  }, [endpoint, locale, script, service, timeOnClient, timeOnServer, timezone]);
```

Comment:
I think this is going to lead to an infinite loop where we keep requesting data from the client - `timeOnClient` is calculated outside of the useEffect hook. The value of this is going to be different on every render, which means the useEffect is going to be called continuously. I just tried adding this to the list of deps and saw this happen in the server logs. We can move the calculation of `timeOnClient` inside the hook and remove it from the deps to fix this :)

Reply:
@tochwill Thanks. This has been fixed now.